apps/frontend/src/hooks/aave/useAaveReservesHistory.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * This hook is used for getting historical reserve data, and it is primarily used with charts.
+ * In particular, this hook is used in the  ̶A̶p̶y̶G̶r̶a̶p̶h̶ AaveReserveOverviewPage (chartJS).
+ */
```

Comment:
Let's remove. They don't do these comments and hook should be generic to be used in multiple places if desired